### PR TITLE
QoL Changes

### PR DESCRIPTION
**Pull request by NotGeri**
This PR includes the changes from #246 and #255 

Here are the main changes:
- [x] Use [`medium-zoom`](https://github.com/francoischalifour/medium-zoom) for nicer image viewing
- [x] Update to the latest Docosaurus version
- [x] Fix not found page
- [x] Fix header image being low quality
- [x] Fix light theme sidebar colours being incorrect
- [x] Simplify document pages
  - [x] Use the regular markdown image syntax
  - [x] Make the header title / image part of the component from the frontmatter
- [x] Fix up/add a nicer docs overview page
- [x] Merge `README.md` & `CONTRIBUTING.md`

Closes:
- #270
- #276
- #278

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,25 +1,196 @@
...
+### Option #1:
+
+You can write an article in a format you prefer and create a support ticket in
+our [Discord server](https://discord.gg/bloom), and it will be converted over for you. 
+Note that documents in Markdown are preferred, however, Microsoft Word and Google Docs files can be accepted,
+and they can be uploaded to the documentation.
+
+---
+
+### Option #2 (Advanced):
```
Comment (zax71) [inline README.md:25]: I would swap these around, I’m sure you prefer people to make PRs

File: README.md [hunk 2]
```diff
@@ -1,25 +1,196 @@
...
+
+You can also contribute to the docs directly by doing the following:
+
+This method makes it easier to import the document(s) and to make sure that the formatting is correct and to make it
+look the way you want it when it is imported.
+
+1. Create a fork of the [GitHub Repository](https://github.com/Bloom-host/BloomDocs)
+2. Make edits to your fork of this repository.
```
Comment (zax71) [inline README.md:33]: Maybe the changes should be made in a separate branch?

File: README.md [hunk 3]
```diff
@@ -1,25 +1,196 @@
...
+
+<details>
+
+<summary>
+
+## 🤖 To BloomBot's Embeds
+
+</summary>
+
+You can also contribute to the embeds used by BloomBot in the Discord where a command is used, and it can provide
```
Comment (zax71) [inline README.md:99]: Add an invite link to the Discord?

File: README.md [hunk 4]
```diff
@@ -1,25 +1,196 @@
...
+
+Please note that changes are not instant and may take up to 24 hours to apply.
+
+---
+
+</details>
+
+## 🦆 Wall Of Gardeners
+These are the awesome people who have contributed at least three guides to our documentation so far!
+If your name is missing, please let us know on [Discord](https://bloom.host/discord)!
```
Comment (zax71) [inline README.md:181]: Can I be added <3

**Review by zax71 — COMMENTED** [review]
Only other comment I have is that alt text should be added on quite a few of the images

**Comment by NotGeri** [comment]
Thank you for the review

> Only other comment I have is that alt text should be added on quite a few of the images

Yes, I initially wanted to go through and add one for each image, but I think it'll be better as a task on its own

I'll address the rest of these really quick

File: README.md (hunk 2 above)
Comment (NotGeri) [reply README.md:33]: > Maybe the changes should be made in a separate branch?

I don't think there is a good way to do that on GitHub. Forking is the intended way to create upstream contrib externally from what I know.

Comment (zax71) [reply README.md:24]: Ah, sorry. I meant forking then making a new branch for changes. That’s how I’ve always been told to do it

Comment (NotGeri) [reply README.md]: Ohh gotcha! Yeah, I added it.

**Comment by NotGeri** [comment]
Going to merge this now. If anyone sees anything strange, please let me know.

We'll follow up on some more accessibility changes later.

**Pull request merged**